template.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strconv"
 	"strings"
 	"time"
+    "code.google.com/p/go.text/unicode/norm"
```

Comment:
Currently this package is not exists, it redirects to https://github.com/golang/go.